src/js/libcs/OpDrawing.js

```diff
@@ -355,7 +355,7 @@ evaluator.drawconic$1 = function(args, modifs) {
         }
 
     }
-    return eval_helper.drawconic(arr, modifs);
+    return eval_helper.drawconic(arr, modifs, "D");
```

Comment:
A similar change is required [in GeoRenderer](https://github.com/CindyJS/CindyJS/blob/v0.8.2/src/js/libgeo/GeoRender.js#L66), else the demo file shows no lines at all.